contrib_bots/lib/poll.py

```diff
@@ -0,0 +1,105 @@
+
+
+class PollHandler(object):
+    '''
+    This plugin facilitates polling.
+    '''
+
+    def __init__(self):
+        # polls is a dict of dict. First dict corresponds to poll title,
+        # second refers to answers in poll
+        self.polls = dict()
+
+        # streams is a convenience object that keeps track of the latest
+        # question for each stream.
+        self.streams = dict()
```

Comment:
Currently only one poll is active per stream (I infer the which question is being responded to by just grabbing the last question for that stream). I think a "close" would certainly be useful. Possibly:
`@close poll title`